packages/demo/src/App.tsx

```diff
@@ -115,6 +103,14 @@ const App = () => {
                             </CustomRoutes>
                         </>
                     ) : null}
+                    <CustomRoutes noLayout>
+                        <Route
+                            path="/custom1"
+                            element={
+                                <CustomRouteNoLayout title="Posts from /custom1" />
+                            }
+                        />
+                    </CustomRoutes>
```

Comment:
Moved this route here as it doesn't make sense to wait for permissions for an anonymous accessible route